### PR TITLE
feat: Add --tool-completion-color for styling command completion

### PR DESCRIPTION
**Pull request by Sinderella**
I had a issue with styling command completion as it's difficult to read for me.

image

Note that I haven't updated the documentation, I am not aware of the guidelines. Not sure if I should edit them directly.

### CODE REVIEW

File: aider/io.py [hunk 1]
```diff
@@ -36,7 +37,14 @@ def __init__(self, items=None):
...
     def __init__(
-        self, root, rel_fnames, addable_rel_fnames, commands, encoding, abs_read_only_fnames=None
+        self,
+        root,
+        rel_fnames,
+        addable_rel_fnames,
+        commands,
+        encoding,
+        abs_read_only_fnames=None,
+        style=None,
```
Comment (jerzydziewierz) [inline aider/io.py:47]: +1 to this style

**Comment by Sinderella** [comment]
I just saw another approach in #1634, seems cleaner, more options.

**Comment by smh** [comment]
sorry, I didn't see your PR when I pushed #1634 🤦 Not sure if mine is cleaner, did notice there are additional completion menus I didn't catch 🤔

**Comment by paul-gauthier** [comment]
Closing in favor of https://github.com/paul-gauthier/aider/pull/1634. Thanks though!

**Pull request closed without merging**